rails/config/routes.rb

```diff
@@ -443,11 +443,13 @@
 
         resources :teacher_classes, only: [:show] do
           member do
-            post :sort
             post :set_active
             post :copy
           end
         end
+        namespace :teacher_classes do
+          post :sort
+        end
```

Comment:
Did you try add this as a collection inside of the teacher_classes resource?
Like 
`get :available_collaborators` above
Perhaps doing that will add some other stuff that will break the route, but if it works, it looks cleaner.

Reply:
Yes, I tried that first and it didn't work so I switched to suing a namespace like the classes route above it uses.

Reply:
Actually I tried it as a bare `post` - I'll try the collection approach.